### DISCUSSION TITLE
[emoji-mart] v5 upgrade

### DISCUSSION THREAD

**Discussion by Scalahansolo** [DefinitelyTyped/DefinitelyTyped#60094] [Issues with a @types package]
Emoji-mart saw a major upgrade recently. (https://github.com/missive/emoji-mart/releases/tag/v5.0.0)

Would be killer to get v5 types setup

[a comment by an automated account is not shown]

**Comment by rugk** [comment]
As [this still applies](https://github.com/missive/emoji-mart/discussions/573#discussioncomment-2663790), maybe better remove or deprecate the emoji-mart @types package?

If you install it it is not really helpful.

**Reply by rugk** [reply]
Ah see also related discussion here: https://github.com/missive/emoji-mart/issues/576
